man/adoc/bpftrace.adoc

```diff
@@ -1916,6 +1916,12 @@ For string arguments in an action block use the `str()` call to retrieve the val
 | raw_smp_processor_id
 | ID of the processor executing the BPF program
 
+| nproc
+| int64
+| n/a
+| n/a
+| The number of CPU processing units online.
```

Comment:
maybe good to add that this is (if i read correctly) a compile time constant and not runtime. If something happens to the cpu count the program has to be restarted?